qiskit_superstaq/custom_gates.py

```diff
@@ -51,13 +60,22 @@ def __array__(self, dtype: Type = None) -> np.ndarray:
             dtype=dtype,
         )
 
+        # sandwiched rx gate commutes and can just be multiplied with non-sandwiched part:
+        return mat @ np.kron(qiskit.circuit.library.RXGate(self.sandwich_rx_rads), np.eye(2))
```

Comment:
should respect `dtype` argument:
```suggestion
        return mat @ np.kron(
            np.asarray(qiskit.circuit.library.RXGate(self.sandwich_rx_rads), dtype=dtype),
            np.eye(2, dtype=dtype),
        )
```